satori/api/client.go

```diff
@@ -161,6 +161,34 @@ func (c *Client) getJsonForAccount(path string, search *string, output interface
 	return nil
 }
 
+func (c *Client) getJsonForAccountWithParams(path string, params *map[string]string, output interface{}) error {
+	req, err := http.NewRequest("GET", fmt.Sprintf("%s%s", c.HostURL, path), nil)
+	if err != nil {
+		return err
+	}
+
+	q := req.URL.Query()
+	for k, v := range *params {
+		q.Set(k, v)
+	}
+	q.Add("accountId", c.AccountId)
+	q.Add("page", "0")
+	q.Add("pageSize", "200")
```

Comment:
pageSize should be much smaller :)

Reply:
It's a generic function and not only for this usage. What do you think should be the size? 
I saw that we set this size also in `getJsonForAccount`